pywt/_extensions/_pywt.pyx

```diff
@@ -372,40 +394,74 @@ cdef public class Wavelet [type WaveletType, object WaveletObject]:
         "Decomposition filters length"
         def __get__(self):
             return self.w.dec_len
+    
+    property family_number:
+        "Wavelet family number"
+        def __get__(self):
+            return self.number
 
     property family_name:
         "Wavelet family name"
         def __get__(self):
-            return self.w.family_name.decode('latin-1')
+            return self.w.base.family_name.decode('latin-1')
 
     property short_family_name:
         "Short wavelet family name"
         def __get__(self):
-            return self.w.short_name.decode('latin-1')
+            return self.w.base.short_name.decode('latin-1')
 
     property orthogonal:
         "Is orthogonal"
         def __get__(self):
-            return bool(self.w.orthogonal)
+            return bool(self.w.base.orthogonal)
         def __set__(self, int value):
-            self.w.orthogonal = (value != 0)
+            self.w.base.orthogonal = (value != 0)
 
     property biorthogonal:
         "Is biorthogonal"
         def __get__(self):
-            return bool(self.w.biorthogonal)
+            return bool(self.w.base.biorthogonal)
         def __set__(self, int value):
-            self.w.biorthogonal = (value != 0)
+            self.w.base.biorthogonal = (value != 0)
+
+    property dwt_possible:
+        "DWT possible"
+        def __get__(self):
+            return True
+
+    property cwt_possible:
+        "CWT possible"
+        def __get__(self):
+            return True
+
+    property lower_bound:
+        "Lower Bound"
+        def __get__(self):
+            if self.w.base.lower_bound != self.w.base.upper_bound:
+                return self.w.base.lower_bound
+        def __set__(self, float value):
+            if self.w is not NULL:
```

Comment:
I don't think `self.w` can ever be `NULL` - this is just from a quick skim though, correct me if I'm wrong here!